res/TensorFlowPythonExamples/examples/segment_sum/__init__.py

```diff
@@ -0,0 +1,5 @@
+import tensorflow as tf
+
+lhs_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(4, 4, 4, 4), name="Hole")
+rhs_ = tf.compat.v1.placeholder(dtype=tf.int32, shape=(4, ), name="Hole")
+op_ = tf.compat.v1.math.segment_sum(lhs_, rhs_)
```

Comment:
(Just out of curiosity) Where does this operator come from?

Reply:
this came from list of https://github.com/Samsung/ONE/issues/1606

Reply:
Oh ;-) Branch new operator that I never saw.